.circleci/config.yml

```diff
@@ -111,8 +107,7 @@ jobs:
       - restore_cache:
           keys:
             - dep-cache-{{ checksum "Gopkg.lock" }}
-      - run: make docker.debug
```

Comment:
I split them so we can see the time - having them in once shot doesn't help. Fine either way